### ISSUE TITLE
Reproduce problem

### ISSUE THREAD

**Issue by ZiJie910811** [ArnabKumarRoy02/ResEmoteNet#12]
Has anyone tried data augmentation like me and still got no good results?

**Comment by zhusiyi601** [comment]
I used oversampling to balance the data on FER2013, but the results did not improve. How did you implement data augmentation?
![Res截图](https://github.com/user-attachments/assets/de58f743-ca6b-434f-ae27-adc894004328)

**Comment by ArnabKumarRoy02** [comment]
Hi @ZiJie910811 @zhusiyi601, this [link](https://drive.google.com/file/d/1Dz27LuCuDRnu172LMy82JWmPDr9nO0dD/view) has the FER2013 dataset that was used in this project. Feel free to use this, hope this helps your problem.

**Comment by ZiJie910811** [comment]
Can I ask you about the actual data amplification method? Because I feel that some of the images I see are generated images, not images from the original data set.

**Comment by ZiJie910811** [comment]
Hello @ArnabKumarRoy02 , I checked your data set and found that your training data set seems to contain a test data set and a validation data set. Is the accuracy of the training correct?

**Comment by ArnabKumarRoy02** [comment]
Hey @ZiJie910811, can you tell me where exactly did you find a test data set and a validation data set in the training set. Since the dataset was augmented to add more samples. Some of the augmentation techniques used were:
 - Random rotations
 - Change in contrast
 - Adding jitter

And I can assure you that none of the images in the dataset and generated images, those were made using the original dataset itself.

**Comment by ZiJie910811** [comment]
Hello @ArnabKumarRoy02 ,sorry, maybe I misunderstood. I want to say sorry to you.
I used the same data amplification method as you, amplifying all images by 1.35 times (not sure how you did it, but the number has been kept as the same as possible)
However, the generalization and accuracy are not as good as your data set, even worse by more than 20%.
